e2eTest/src/main/java/com/google/android/ground/e2etest/SurveyRunnerTest.kt

```diff
@@ -0,0 +1,235 @@
+/*
```

Comment:
Can we reuse the `androidTest` package instead of creating a new one? Or are these tests somehow fundamentally different from normal instrumented tests?

Reply:
I put them in their own module following recommendation from the [Android dev guide](https://developer.android.com/studio/test/advanced-test-setup#use-separate-test-modules-for-instrumented-tests):

> If you want to have a dedicated module for instrumented tests, to isolate the rest of your code from your tests, create a separate test module and configure its build similar to that of a library module.

The nice thing about this setup is that they won't accidentally be included in the main module's unit tests, since the tests don't work without the web emulator also running in the background, plus we don't have to keep track of filtering them out of the main module's test.

I don't know why, but they recommend putting the tests in `main` instead of `connectedAndroidTest` which would be customary for instrumented tests.